spec/requests/locale_detection_request_spec.rb

```diff
@@ -57,26 +50,29 @@
 
     context "given multiple detected locales" do
       include_context :request_spec_logged_in_as_user
-      before do
-        allow(Flipper).to receive(:enabled?).with(:localization, current_user).and_return(true)
-      end
 
       it "gives highest precedence to query param" do
-        current_user.update(preferred_language: :es)
+        current_user.update_attribute :preferred_language, :es
```

Comment:
A bit early to worry about this, but just a [head's up](https://github.com/rails/rails/pull/31998):

> DEPRECATION WARNING: update_attributes! is deprecated and will be removed from Rails 6.1 (please, use update! instead)

Reply:
Also, `update_attribute` (sans -_s_) bypasses validations, not clear if / why that's intended here.

Reply:
That is intended here. Because `:es` isn't an official language